src/pages/minutes.js

```diff
@@ -17,7 +17,10 @@ export default function Minutes({ data }) {
 				<div className="downloads">
 					<h3>Downloads:</h3>
 					{nodes.map(node => (
-						<Link  key={node.id} className={"download-link " + node.node.extension} href={node.node.publicURL} download>{node.node.name}.{node.node.extension} - {node.node.size / 100}kb</Link>
+						<a key={node.node.Minutes.id} className={"download-link " + node.node.Minutes.localFile.extension} href={node.node.Minutes.localFile.publicURL} download>
+							{node.node.Minutes.name}.{node.node.Minutes.extension} - {node.node.Minutes.size / 100}kb
```

Comment:
Works great but something is adding a full stop after the file extension. Text has an id `#text` but can't find any ref to it anywhere else. `node.node.Minutes.localFile.extension` doesn't come though with the full stop either. Not enough of a problem but putting this note here for future reference.

Reply:
Wrap line 21 in a `<div>` as with line 22. Prevents the file info turning default blue on hover. Will push changes after finishing the review.